pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "open-rarity"
-version = "0.4.0-beta"
-description = "Open-Rarity library is an open standard that provides an easy , explanable and reproducible computation for NFT rarity"
+version = "0.5.0-beta"
```

Comment:
i think there is no major interface change - should be 0.4.1 ?

Reply:
Will change

Reply:
I did minor because it is different behavior though